salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -138,13 +138,18 @@
 
                 // specify allowed elements, attributes, and classes
                 // this must conform to the `STANDARD_MAPPING`
-                var filter = ' p em strong strike u sub sup; a[!href](salsah-link)';
+                var filter = ' p em strong strike u sub sup hr h1 h2 h3 h4 h5 h6 pre table tbody tr td ol ul li cite blockquote code; a[!href](salsah-link) ';
 
 				var config = {
 					language: (SALSAH.userprofile && SALSAH.userprofile.userData && SALSAH.userprofile.userData.lang) ? SALSAH.userprofile.userData.lang : 'en' ,
                     allowedContent: filter,
                     pasteFilter: filter,
+                    format_tags: 'p;h1;h2;h3;h4;h5;h6;pre',
```

Comment:
- format_tags
`format_tags` is the list of element that appears in the `format` drop button:
https://docs.ckeditor.com/#!/api/CKEDITOR.config-cfg-format_tags  
I'll add this link too.  
I just removed `address;div` from the default value as the former is unknown from our mapping and the latter requires more details to be parsed.

- Toolbar configuration
yes, I used this tool to *bake* the proposed config.  
Looking at it again, I see the *advanced* mode that allows to specify the individual items and not the groups that you are not so found of: https://ckeditor.com/latest/samples/toolbarconfigurator/index.html#advanced  
is it that link that you wanted to point to?

- default values `h4`, `h5`, `h6`
I don't understand this point...